src/frontend/components/TranscriptReader/TranscriptReader.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Paragraph, Text } from 'grommet';
+import * as React from 'react';
+import { VTTCue, WebVTT } from 'vtt.js';
+
+import { TimedTextTranscript } from '../../types/tracks';
+import { TranscriptSentence } from '../TranscriptSentence/TranscriptSentence';
+
+interface TranscriptReaderProps {
+  transcript: TimedTextTranscript;
+  currentTime: number;
+}
+
+interface TranscriptReaderState {
+  cues: VTTCue[];
+}
+
+export class TranscriptReader extends React.Component<
+  TranscriptReaderProps,
+  TranscriptReaderState
+> {
+  constructor(props: TranscriptReaderProps) {
+    super(props);
+
+    this.state = {
+      cues: [],
+    };
+  }
+
+  async componentDidMount() {
+    const parser = new WebVTT.Parser(window, WebVTT.StringDecoder());
+
+    const response = await fetch(this.props.transcript.url);
+    const content = await response.text();
+    parser.oncue = (cue: VTTCue) => {
+      this.setState(prevState => {
```

Comment:
`this.setState({ cues: [...this.state.cues, cue] })`

Reply:
done